From 8c482683edf6aee17d4718150158293461b88b3b Mon Sep 17 00:00:00 2001
From: Michael Stahl <mstahl@redhat.com>
Date: Mon, 20 Aug 2012 10:41:08 +0000
Subject: configure: fix S390X CPUNAME:

The value was changed to S390x, presumably by accident, breaking the
build (regression from 480b32d3809830002ecc28508d40af3ecd4ad272).

Change-Id: Icca90818c6363855437a273714a8317becdfd0e5
---
diff --git a/configure.in b/configure.in
index c47a5fe..6f8d115 100644
--- a/configure.in
+++ b/configure.in
@@ -3459,7 +3459,7 @@ linux-gnu*)
         ;;
     s390x)
         CPU=3
-        CPUNAME=S390x
+        CPUNAME=S390X
         RTL_ARCH=S390x
         LIB64="lib64"
         OUTPATH=unxlngs390x
--
cgit v0.9.0.2-2-gbebe
